charts/clusternet-controller-manager/values.yaml

```diff
@@ -0,0 +1,138 @@
+## @section Common parameters
+
+## @param kubeVersion Override Kubernetes version
+##
+kubeVersion: ""
+## @param nameOverride String to partially override common.names.fullname template (will maintain the release name)
+##
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname template
+##
+fullnameOverride: ""
+## @param commonLabels Labels to add to all deployed objects
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+
+## @section Clusternet Parameters
+## @param replicaCount Number of clusternet-controller-manager replicas desired
+##
+replicaCount: 3
+serviceAccount:
+  ## @param serviceAccount.name The name of the ServiceAccount to create
+  ## If not set, a name is generated using the common.names.fullname template
+  name: "clusternet-controller-manager"
+## @param securePort Port where clusternet-controller-manager will be running
+##
+securePort: 443
+
+## @param controller a list of controllers to enable
+##
+controller: "*"
+## @param image.registry clusternet-controller-manager image registry
+## @param image.repository clusternet-controller-manager image repository
+## @param image.tag clusternet-controller-manager image tag (immutable tags are recommended)
+## @param image.pullPolicy clusternet-controller-manager image pull policy
+## @param image.pullSecrets Specify docker-registry secret names as an array
+##
+image:
+  registry: ghcr.io
+  repository: clusternet/clusternet-controller-manager
+  tag: v0.14.0
```

Comment:
The image is targeted to `v0.15.0`.

Reply:
also remove the extra parameters in hub and upgrade to v0.15.0